.github/workflows/run_examples_local.yml

```diff
@@ -8,8 +8,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6]
-        os: [ubuntu-18.04]
+        python-version: [3.6, 3.7, 3.8]
+        os: [ubuntu-latest, macos-latest]
```

Comment:
This setup still does not run at macOS at all. I suggest we drop it from the os list and only run ubuntu as earlier...

Reply:
Also, it was made into `ubuntu-18.04` because that is the latest OS release with a ppa for OPM-flow. Please check that it is available for for 20.04 before you change this, as Github Actions are at the moment migrating `ubuntu-latest` from `18.04` to `20.04` and we will be hit by this at some random point if not...

Reply:
> This setup still does not run at macOS at all. I suggest we drop it from the os list and only run ubuntu as earlier.

it does run polynomial demo on macOS

Reply:
> Also, it was made into `ubuntu-18.04` because that is the latest OS release with a ppa for OPM-flow. Please check that it is available for for 20.04 before you change this, as Github Actions are at the moment migrating `ubuntu-latest` from `18.04` to `20.04` and we will be hit by this at some random point if not...

I will change this back to 18.04